### ISSUE TITLE
Web app: Unable to navigate to the note editing area with screen reader

### ISSUE THREAD

**Issue by reginabally** [Automattic/simplenote-electron#2365]
### Expected
I would expect to be able to navigate to the note editing area to edit notes with a screen reader (NVDA).

### Observed

The cursor skipped the note editing area from the search bar and moved to the tags section.

### Reproduced

*Exact steps shared in 3354940-zen*

1. Open Simplenote web app (https://app.simplenote.com/) 
2. Find and open one note. 
3. Press e to move to the next edit field. (Here user could search their notes.) 
4. Press e to move to the next edit field. (In the previous version, here the user could modify their notes.)
5. The cursor moves to the next edit field (added tag).

### Where did you see the bug

- OS: Windows
- OS version: 10.0.18363 
- Browser (if applicable): Mozilla Firefox  
- Browser version (if applicable): 81.0 (x64 zh-TW)
- Simplenote app version: https://app.simplenote.com
- The version of the NVDA screen reader: 2020.2

Reported in 3354940-zen.

**Comment by msilbers** [comment]
Likely related to #1935

**Comment by surfer0627** [comment]
Hi,
Is it possible to consider to add an option, like we did for simplenote beta?

So, screen reader users could switch to previous version, and edit their notes.

Or, could sighted users share steps "how to navigate to the note editing area"?

Thanks.

**Comment by codebykat** [comment]
@surfer0627 As a workaround, you should be able to use the shortcut Ctrl+Shift+Y to jump to the note editing area. (This shortcut toggles between editing tags and editing the note, but if neither is currently focused, it will move the cursor to the note.) Hope this helps!

**Comment by codebykat** [comment]
From reading the description, this looks like yet another piece of functionality we've lost in the trade-off of Monaco editor (which does not use editable text elements under the hood, so browsers, browser extensions, and -- seemingly -- screen readers are unable to recognize these areas as text areas). This is likely extremely complex, if not impossible, to fix. 😭

**Comment by surfer0627** [comment]
@codebykat
Shortcut Ctrl+Shift+Y works as expected (navigate to the note editing area).

In this way, I can edit note.

Thanks.
